internal/config/notify/parse.go

```diff
@@ -58,13 +58,24 @@ func TestSubSysNotificationTargets(ctx context.Context, cfg config.Config, subSy
 		defer target.Close()
 	}
 
+	tgts, ok := ctx.Value(config.ContextKeyForTargetFromConfig).(map[string]bool)
+	if !ok {
+		tgts = make(map[string]bool)
+	}
 	for _, target := range targetList {
-		yes, err := target.IsActive()
-		if err == nil && !yes {
-			err = ErrTargetsOffline
-		}
-		if err != nil {
-			return fmt.Errorf("error (%s): %w", target.ID(), err)
+		if tgts[target.ID().ID] {
+			// When target set should be online
+			yes, err := target.IsActive()
+			if err == nil && !yes {
+				err = ErrTargetsOffline
+			}
+			if err != nil {
+				return fmt.Errorf("error (%s): %w", target.ID(), err)
+			}
+		} else {
+			// Just for call init.
+			// Ignore target is online or offline
+			_, _ = target.IsActive()
```

Comment:
IMO, We don't need to ping other targets too. This is a side effect and can be avoided. 

`Set|Delete` don't need to care about Init.

Need your help here to decide @harshavardhana